src/App.js

```diff
@@ -50,15 +47,16 @@ class App extends React.Component {
 
 				<DailyTotal/>
 
-				<SettingsDrawer
+				<ConnectedSettingsDrawer
 					isOpen={settingsDrawerIsOpen}
 					onClose={this.toggleSettingsDrawer}
+					heading="Settings"
+
+					trustSettingValue="1"
+					onSave={this.toggleSettingsDrawer}
 				>
```

Comment:
We don't need `<X></X>` here as there's no child elements, this can be `<X/>`

Reply:
Fixed